src/net/tcp.rs

```diff
@@ -618,24 +629,24 @@ impl Evented for TcpListener {
  *
  */
 
-#[cfg(unix)]
+#[cfg(all(unix, not(target_os = "fuchsia")))]
 use std::os::unix::io::{IntoRawFd, AsRawFd, FromRawFd, RawFd};
 
-#[cfg(unix)]
+#[cfg(all(unix, not(target_os = "fuchsia")))]
```

Comment:
It is, but we want to keep that hidden-- eventually we want to move away from the file-descriptor-based approach to something built on the native channels. That interface isn't ready to use, yet, though.